tests/classification_test.py

```diff
@@ -0,0 +1,118 @@
+from x_transformers import TransformerWrapper, Encoder
+import torch
+from torch import nn
+
+# CLS token test
+transformer = TransformerWrapper(
```

Comment:
do you want to try moving all of this under a function like `def test_classification():` ?

Reply:
you can just slap it all into `test_x_transformers.py`

Reply:
Sure, I'll do that. Additionally, I'm trying to make an example section for IMBD dataset classification + minor fixes